installer/src/main/bin/createDomain.cmd

```diff
@@ -120,6 +124,18 @@ ECHO         variable_file   - the location of the property file containing the
 ECHO                           the model. This can also be specified as a comma-separated list of property files,
 ECHO                           where each successive set of properties layers on top of the previous ones.
 ECHO.
+ECHO         passphrase_env  - An alternative to entering the encryption passphrase at a prompt. The value is a
+ECHO                           ENVIRONMENT VARIABLE name that WDT will use to retrieve the passphrase.
+ECHO.
+ECHO         passphrase_file - An alternative to entering the encryption passphrase at a prompt. The value is a
+ECHO                           the name of a file with a string value which WDT will read to retrieve the passphrase.
+ECHO.
+ECHO         opss_wallet_passphrase_env  - An alternative to entering the opss wallet passphrase at a prompt. The value is a
```

Comment:
*OPSS (in description) in two places

Reply:
you want to remove OPSS or change it to upper case?

Reply:
Just upper case